x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
+            }
+          }
+        },
+        {
+          "cloud_provider": {
+            "path_match": "cloud.provider",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "cloud_availability_zone": {
+            "path_match": "cloud.availability_zone",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "cloud_region": {
+            "path_match": "cloud.region",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "host_hostname": {
+            "path_match": "host.hostname",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "host_name": {
+            "path_match": "host.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "container_id": {
+            "path_match": "container.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "container_name": {
+            "path_match": "container.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_namespace": {
+            "path_match": "orchestrator.namespace",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_id": {
+            "path_match": "orchestrator.cluster.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_id": {
+            "path_match": "orchestrator.cluster.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_resource_id": {
+            "path_match": "orchestrator.resource.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "orchestrator_resource_name": {
+            "path_match": "orchestrator.resource.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        }
+      ],
+      "dynamic": "runtime",
+      "date_detection": false,
       "properties": {
+        "@timestamp": {
+          "type": "date",
+          "ignore_malformed": false
```

Comment:
If `@timestamp` is malformed, the `_logs` endpoint handles this error and ingests the data again?

Reply:
Yes, events with a malformed timestamp will land in the dlq. The dlq always sets the top-level `@timestamp` to the ingest time. The `@timestamp` from the even is stored in `event.original.@timestamp`.